.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)